lib/game.rb

```diff
@@ -13,16 +13,41 @@ def initialize(input:, output:)
   end
 
   def play
+    introduce_the_game
+    play_to_completion
+    bid_farewell
+  end
+
+  private
+
+  attr_reader :grid, :input, :output
+
+  def introduce_the_game
     grid_view.render
+  end
+
+  def play_to_completion
```

Comment:
Hmm, looking at this now I think I want to extract all these main loop things into another class...